src/osemgrep/cli_scan/Scan_CLI.ml

```diff
@@ -154,6 +164,21 @@ let o_version_check : bool Term.t =
  *  rules' languages. These options alter which files Semgrep scans."
  *)
 
+let o_project_root : string option Term.t =
```

Comment:
please add a big comment like (* NEW: This is a new option! it's not in legacy semgrep *).
I actually put the new flags in another section below.